liquid_node/backup.py

```diff
@@ -258,18 +272,26 @@ def backup_collection_es(dest, name):
             else:
                 raise RuntimeError("Something went wrong: %r" % snapshot)
         log.info(f"Snapshot done in {int(time()-t0)}s")
-
-        backup_files(tmp_file, f"/es_repo/backup-{name}", [], SNOOP_ES_ALLOC)
+        if es_url == '/_h_es':
+            backup_files(tmp_file, f"/es_repo/backup-{name}", [], HYPOTHESIS_ES_ALLOC)
+        else:
+            backup_files(tmp_file, f"/es_repo/backup-{name}", [], SNOOP_ES_ALLOC)
 
         dest_file = dest / "es.tgz"
         tmp_file.rename(dest_file)
     finally:
         es.delete(f"/_snapshot/backup-{name}/snapshot")
         es.delete(f"/_snapshot/backup-{name}")
-        rm_cmd = (
-            f"./liquid dockerexec {SNOOP_ES_ALLOC} "
-            f"rm -rf /es_repo/backup-{name} "
-        )
+        if es_url == "/_h_es":
```

Comment:
Same as above: what if we add another elasticsearch here? We'd need more `if` branches